internal/schema/schemas/posit-publishing-record-schema-v3.json

```
@@ -70,14 +70,10 @@
       "description": "Date and time the deployment was created.",
       "examples": ["2024-01-19T09:33:33.131481-05:00"]
     },
-    "local_id": {
-      "type": "string",
-      "description": "ID of active/latest deployment sequence."
-    },
-    "aborted_at": {
+    "dismissed_at": {
       "type": "string",
       "format": "date-time",
-      "description": "Date and time that the current deployment flow was aborted. Is empty if flow was not aborted.",
+      "description": "Date and time that the current deployment flow was dismissed. Is empty if flow was not dismissed.",
```

I should have suggested this on the original wording. This makes a bit more sense to me, but feel free to ignore.

```suggestion
      "description": "Date and time that the deployment process was dismissed. Is empty if the deployment process was not dismissed.",
```

If changed here the draft should also have this change.

---

Changed both to be your recommendation. Thanks.